### DISCUSSION TITLE
Clean store for going live procedure

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [OpenMage/magento-lts#1967] [Questions & Answers]
What is the advised way of cleaning up magento after the testing period for going live. For instance I would like to delete testing orders, abandoned  carts etc.

**Comment by addison74** [comment]
If you search on Internet you will find on Stackexchange tons of articles related to delete testing products, customers, orders, price/stock alerts, wishlists, and many more, before going with a store in production.

In fact it is just an SQL script for doing that. You can also investigate by your own the database and decide what data to keep and what data to delete (truncate). This is how I learnt about every table Magento is using to store data.

**Reply by f1-outsourcing** [reply]
The reason why I am asking here is to try and benefit from experience.

there is already huge difference between these posts.
https://stackoverflow.com/questions/4526914/how-can-i-delete-test-order-from-magento
https://community.magento.com/t5/Magento-1-x-Technical-Issues/Delete-Orders/td-p/101544

**Comment by addison74** [comment]
The two links refer to a part of the database that has over 300 tables. 

I recommend that you do an OpenMage installation from scratch and then export the database. Make a comparison between the full and empty databases. This way you will notice how many tables are used. Once you have identified the tables you want to truncate, you can make your own SQL script to run in Terminal. 

As I mentioned in the previous post if you want to keep only the categories and products, I have to delete a lot of information such as catalog searches, price and stock alerts, wish lists, tags, reviews, newsletter subscription, all sales tables, customers, log tables, maybe terms and conditions, reset Orders/Shipment/Invoices ID's. 

You can also export only the tables related to products and categories and import them to a "freshly installed" database. Of course paying attention to other tables such as attributes, attribute sets, store configuration, extensions/modules. An analysis of the database and of each table will make you understand how the data part works. There are situations when you will end up modifying information directly in the table, to empty certain tables, delete rows of spam data.
